schema/alexandria_convention/Metadata_convention_releases.txt

```diff
@@ -0,0 +1,14 @@
+2.2.1
```

Comment:
This looks useful!  Suggestions:

* Add dates to releases, to aid debugging
* Use Markdown, to enable clickable links in GitHub via e.g. `[Zendesk #298615](https://broadinstitute.zendesk.com/agent/tickets/298615)`
* Make file name lower case like other files.  Underscore-delimited title case file names are rare in SCP repos.

Reply:
I've updated the file `52a8e8e`, great suggestions - its much more readable and useful now.